python/cugraph/tests/utils.py

```diff
@@ -37,41 +37,40 @@
 CUPY_MATRIX_TYPES = [cp_coo_matrix, cp_csr_matrix, cp_csc_matrix]
 SCIPY_MATRIX_TYPES = [sp_coo_matrix, sp_csr_matrix, sp_csc_matrix]
 
+RAPIDS_DATASET_ROOT_DIR = os.getenv("RAPIDS_DATASET_ROOT_DIR", "../datasets")
+RAPIDS_DATASET_ROOT_DIR_PATH = Path(RAPIDS_DATASET_ROOT_DIR)
```

Comment:
I am just wondering why you replaced `PurePath` with `Path`. My understanding so far of `RAPIDS_DATASET_ROOT_DIR` is to allow the tests to find the datasets (no systems call). But `Path` provide method to do systems call on path object. Are we doing systems call or planning to do so?

Reply:
I was thinking we might have test or setup code that wants to do things like `assert RAPIDS_DATASET_ROOT_DIR.is_dir()` or possibly `if (RAPIDS_DATASET_ROOT_DIR / "foo").exists(): ...`, which aren't available on a `PurePath` object.

Reply:
Make sense